.github/bin/tag

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+[ "$#" -ge "1" ] || { echo "usage: $0 v1.0.11"; exit 1; }
+
+set -x
+
+tag=$1
+
+git tag --delete $tag
+git push origin :$tag
```

Comment:
I'm not sure that blindly deleting an existing tag is a good idea; wouldn't it be better to error out in the case that a tag already exists? If you delete and recreate it, it'll force a new release, which would overwrite existing release binaries, which might've already been downloaded by users.

More broadly though, I'm not sure this script adds a lot of value. Creating and pushing a tag is easy enough to do manually, and those looking at a simpler way could just use the GitHub UI.

Reply:
Removed. Useful for testing releases.